db/migrate/20190514203351_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[5.2]
+  def self.up
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :username, limit: 50, format: { with: /^[a-zA-Z]+$/ }, uniqueless: false, null: false
```

Comment:
You don't have to add the same validations in the DB as in the model. Here you can just add: `null: false`. Also, the DB won't recognize some validations, like `format`.